lambda_code/src/models/fhir_immunization_post_validators.py

```diff
@@ -439,37 +397,29 @@ def validate_report_origin_text(cls, values: dict) -> dict:
         )
 
         # Handle conditional mandation logic
-        mandation = vaccine_type_applicable_validations["report_origin_text"][
-            cls.vaccine_type
-        ]
+        mandation = vaccine_type_applicable_validations["report_origin_text"][cls.vaccine_type]
         if not values["primarySource"]:
             mandation = Mandation.mandatory
 
         check_mandation_requirements_met(
             field_value=report_origin_text,
             field_location=field_location,
             mandation=mandation,
-            bespoke_mandatory_error_message=f"{field_location} is mandatory when "
-            + "primarySource is false",
+            bespoke_mandatory_error_message=f"{field_location} is mandatory when " + "primarySource is false",
```

Comment:
Doesn't need the " + "

Reply:
this is what happens when you change your auto formatter to be a longer length. removed.